### ISSUE TITLE
Crash when config value should be an array

### ISSUE THREAD

**Issue by kohenkatz** [ash-jc-allen/laravel-config-validator#47]
In my application config, I have a config item that needs to be an array of values, with at least one value in it:

```
'names' => array_filter(explode(',', env('NAMES', ''))),
```

I wanted to create a validation rule for it like this:

```
Rule::make('migrate_from')->rules(['array', 'min:1']),
```

When I run `config:validate`, I get this exception:

```

   Illuminate\View\ViewException 

  htmlspecialchars(): Argument #1 ($string) must be of type string, array given (View: /path/to/app/vendor/ashallendesign/laravel-config-validator/resources/views/validate-config.blade.php)

  at /path/to/app/vendor/illuminate/support/helpers.php:119
    115▕         if ($value instanceof Htmlable) {
    116▕             return $value->toHtml();
    117▕         }
    118▕
  ➜ 119▕         return htmlspecialchars($value ?? '', ENT_QUOTES, 'UTF-8', $doubleEncode);
    120▕     }
    121▕ }
    122▕
    123▕ if (! function_exists('env')) {

  1   /path/to/app/vendor/illuminate/view/Engines/PhpEngine.php:60
      Illuminate\View\Engines\CompilerEngine::handleViewException(Object(TypeError))

  2   /path/to/app/vendor/illuminate/support/helpers.php:119
      TypeError::("htmlspecialchars(): Argument #1 ($string) must be of type string, array given")
```

I traced this to this line in the view:

```
<b>{{ config($configField) }}</b>
```

To make it work, I replaced that line with this:

```
<b>{{ json_encode(config($configField, JSON_PRETTY_PRINT)) }}</b>
```

I'm not sure whether my fix is a good way to do it (and I would submit it as a PR), or whether there is a better way. On one hand, this way will always work and will show a nice output for all kinds of config values (including simple and associative arrays). On the other hand, it changes how regular string values appear (by putting quotes around them).

Thoughts?

**Comment by ash-jc-allen** [comment]
Hey @kohenkatz! Sorry that you're having trouble with the package. I hope that it's been useful apart from the error?

Thanks for the really detailed issue, you've made life a lot easier for me. I've put together a PR (#48) using your fix that should hopefully fix the bug. If you get the chance, it'd be really helpful if you could give it a quick run on your end (no problem if not though). If you think it's all okay, I'll get it merged in and released ASAP for you 😄

**Comment by kohenkatz** [comment]
@ash-jc-allen That PR looks great! Thanks for the quick fix.

At some point soon, I'm planning to open a PR here to help [Laravel Zero](https://www.laravel-zero.com) users use this package too.

**Comment by ash-jc-allen** [comment]
This has now been released as part of v2.2.0 (https://github.com/ash-jc-allen/laravel-config-validator/releases/tag/v2.2.0)

I'd be more than happy to look at a PR for this. But, I've not used Laravel Zero in a while, so I might need a hand from someone else with reviewing it :)
